src/test/java/visualizations/ScatterChartTest.java

```diff
@@ -0,0 +1,74 @@
+package visualizations;
```

Comment:
same with this PR: https://github.com/jethronap/JavaDataAnalysis/pull/12 currently I don't think these tests put any value for us...the case is covered by the examples or no?

Reply:
removed tests.